File: src/main/java/com/matamercer/microblog/web/error/RevokedRefreshTokenException.java
package com.matamercer.microblog.web.error;

public class RevokedRefreshTokenException extends  RuntimeException{
        private static final long serialVersionUID = 5861310537366287163L;

        public RevokedRefreshTokenException() {
            super();
        }

        public RevokedRefreshTokenException(final String message, final Throwable cause) {
            super(message, cause);
        }

        public RevokedRefreshTokenException(final String message) {
            super(message);
        }

        public RevokedRefreshTokenException(final Throwable cause) {
            super(cause);
        }

}

File: src/main/java/com/matamercer/microblog/web/error/exceptions/RevokedRefreshTokenException.kt
package com.matamercer.microblog.web.error.exceptions

class RevokedRefreshTokenException : RuntimeException {
    constructor() : super()
    constructor(message: String?, cause: Throwable?) : super(message, cause)
    constructor(message: String?) : super(message)
    constructor(cause: Throwable?) : super(cause)

    companion object {
        private const val serialVersionUID = 5861310537366287163L
    }
}